client/blocks/daily-post-button/docs/example.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import DailyPostButton from 'blocks/daily-post-button';
+import { dailyPromptPost } from 'blocks/daily-post-button/test/fixtures';
+
+const DailyPostButtonExample = () => {
+	return (
+		<div className="design-assets__group">
+			<DailyPostButton post={ dailyPromptPost } />
+		</div>
+	);
+};
+
+DailyPostButtonExample.displayName = 'DailyPostButton';
```

Comment:
What does this line do? Doesn't the displayName get set automatically? [source]( https://facebook.github.io/react/docs/react-component.html#displayname)